lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+
+class Menu 
+    
+    attr_reader :dishes
+
+    def initialize(dishes)
+        @dishes = dishes
+    end
+
+    def print
+        dishes.map do |title, price|
+        "%s £%.2f" % [title.to_s.capitalize, price]
```

Comment:
Readability may be increased if string interpolation was used here instead of format strings